app/main.js

```diff
@@ -85,26 +85,39 @@ if (!gotTheLock) {
     }
 
     if (!cmd.checkInMain()) {
-      log.info(`Stretchly: command '${cmd.command}' executed in second-instance, dropped in main instance`)
+      log.info(`Stretchly: command '${cmd.command}' executed in second instance, dropped in main instance`)
       return
     }
+
     switch (cmd.command) {
       case 'reset':
-        log.info('Stretchly: reseting breaks (requested by second instance)')
+        log.info('Stretchly: resetting breaks (requested by second instance)')
         resetBreaks()
         break
 
-      case 'mini':
+      case 'mini': {
         log.info('Stretchly: skip to Mini Break (requested by second instance)')
+        const ms = cmd.waitToMs()
+        if (ms === -1) {
```

Comment:
can we rename `ms` to `delay`? It would be better readable

Reply:
No worries. I'll make the `pause` code clearer while I'm at it, given that's where I ~~stole~~ borrowed this from :)